app/policies/partnership_policy.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class PartnershipPolicy < ApplicationPolicy
+  def update?
+    user&.induction_coordinator? && user&.schools&.include?(record.school)
```

Comment:
```
return false unless user&.induction_coordinator?
user.schools.include?(record.school)
```

What about admin users?

Reply:
Until we have a way to do that, I think this is fine. There might some logic around what admins are able to do, and that would be part of implementing their functionality

Reply:
Lead providers will also need to be able to create, but same applied